Bz1525328: manageiq-automate role

cbudz opened the pull request:
Hi Dayle,

Would you mind having  a look at this PR? I've gotten some feedback from Drew, who designed the role, which you can see in the BZ. Please let me know what you think. 

Thanks,
Chris

dayleparker commented:
Hi Chris,
The content looks really good. I just noticed there might be a bit of markup missing for system items (not sure what to call extra_vars, etc.? ... a method?), mostly in the Role Variables section, but also for the filename "requirements.yml". I'd suggest marking these terms up with `  or _ for consistency, and so they're clear not to be translated. What is our usual convention for this? Just wanted to check, as maybe it's different for Ansible...
A few last nits - there is a | character at the end of the "manageiq_validate_certs" sentence, which I think needs removing, and would it be correct to change "the playbook vars" to "the playbook variables"?
Please let me know what you think...
Thanks Chris!

cbudz commented:
Hi Dayle,

Thanks for the feedback. I've gone ahead and revised the doc based on the items you've highlighted, and made a few additional changes. I'll itemize here for your review:

- Added markup for variable names. _auto_commit_, _manageiq_valiadate_certs, and _extra_vars_ a all been italicized per IBM style guide. I reviewed the Ansible documentation to see how they treated 'extra vars't but there's no markup applied. My opinion is that it does abide by IBM guide's defintion of a variable name, so I've marked it up accordinly. 

- File and directory names have been marked up according to IBM.

- Removed the pipe character at the end of the sentence. Also in this sentence I removed reference to the `manageiq_connection` variable, since that doesn't apply to playbooks run natively on an CloudForms appliance. I should have removed that when I removed the description of it as a variable.

- Moved a sentence in the note, to appear after the code sample. It made more sense to me reading it this way. 

- Changed playbook to playbook variables. 

- Corrected a few sundry items I should have caught this afternoon before I submitted my PR. 

I think that covers everything. Preview is updated with my changes. 

-Chris

dayleparker commented:
Heya @cbudz,
Thanks for the summary, the new preview, and for making those edits -- it all looks great and reads well. Merging it :)
Cheers,
Dayle
